node/packages/parser/src/ethereum/thor.ts

```diff
@@ -67,14 +69,21 @@ export class Parser {
     // We didn't recognise the sigHash - exit
     if (!result) return
 
+    const decoded = this.abiInterface.parseTransaction({ data: txData })
```

Comment:
Something that we can address on a follow up pr, but we can probably clean up how we are doing `decodeFunctionData` vs `parseTransaction`. Don't see why we can't just use `parseTransaction` at this point so we have both method name and args and then just have a condition to return on unsupported methods

Reply:
Fixed up in https://github.com/shapeshift/unchained/pull/267/commits/93a84e28422f0cc76c1ebddfbccfac54883b5310